packages/eslint-plugin-pf-codemods/test/rules/v5/pagination-rename-props.js

```diff
@@ -114,5 +110,65 @@ ruleTester.run("pagination-rename-props", rule, {
         },
       ],
     })),
+    ...Object.keys(updatedTitlesPropNames).map((oldName) => ({
+      code: `import * as React from "react";
```

Comment:
Any reason for including the import from react?

Reply:
just copied it from the file I am testing the codemod on, I keep it there to not have ugly red errors, but it is unnecessary in the test, removing it now